### PR TITLE
Feat/daemon spawn gui

### PR DESCRIPTION
**Pull request by coryschwartz**
# Goals

Allow spawning from the GUI

# Implementation

On top of Cory's PR, I've added some UI cleanup + a couple bug fixes. I also made schedule task the default accordion pane to start open (since task list is super long)

dealbotspawndemo

### CODE REVIEW

File: controller/app/index.js [hunk 1]
```diff
@@ -77,21 +79,65 @@ function gotTable(data) {
...
-            {title:'Delete', field: 'operate', align: 'center', formatter: operate, events: { 'click .remove': cancel}}
+            {title:'ID', field:'id'},
+            {title:'Region', field:'region'},
+            // {title:'Wallet', field:'wallet', formatter: stringify},
+            {title:'Wallet', field:'wallet.address'},
         ],
-        data: processedData,
+        data: [data],
     });
-    firstLoad = false
```
Comment (willscott) [inline controller/app/index.js:91]: where is this getting set to false now?

Comment (hannahhoward) [reply controller/app/index.js:91]: it's just up near the top of the file.

Comment (hannahhoward) [reply controller/app/index.js:91]: nvm this is actually an issue we need to address.

File: controller/app/index.js [hunk 2]
```diff
@@ -77,21 +82,57 @@ function gotTable(data) {
...
+        firstLoad = false
+    } else {
+        $("#taskTable").bootstrapTable('load', processedData)
+    }
+}
+
+function gotRegions(data) {
+    regionList = data["regions"]
+    $.each(regionList, (reg) => {
+        $('<button/>')
```
Comment (willscott) [inline controller/app/index.js:105]: maybe this is a radio choice? you should be only able to choose 1 of them at a time, right?

File: controller/static/index.html [hunk 3]
```diff
@@ -124,15 +122,76 @@ <h2 class="accordion-header" id="headingBots">
...
+                                <div id="botMinFil">
+                                    <label for="newBotMinFil" class="form-label">Minimum Fil: (optional)</label>
+                                    <input class="form-control" type="number" id="newBotMinFil" value="0">
+                                </div>
+
+                                If you would like to provide a wallet, add details below.
+                                Otherwise, a wallet will be new wallet will be generated for you.
+                                When wallets are generated for you, the private key will not be
+                                visible on this page.
+                                <div id="botWalletAddress">
```
Comment (willscott) [inline controller/static/index.html:175]: can we not calculate address from the wallet export?

**Review by willscott — APPROVED** [review]

File: controller/app/index.js (hunk 2 above)
Comment (hannahhoward) [reply controller/app/index.js:105]: I think this behaves like a radio. Also it looks nice :)

**Pull request merged**